src/eduid/satosa/scimapi/serve_static.py

```diff
@@ -55,8 +55,8 @@ def _handle(self, context):
                 response = f.read()
                 mimetype = mimetypes.guess_type(file)[0]
                 logger.debug(f"mimetype {mimetype}")
-        except IOError:
-            response = "File not found".encode()
+        except OSError:
+            response = b"File not found"
```

Comment:
linting said .encode() was unnecessary for ut-8 encoding but it was there to make it a bytes string literal which can be done with b-strings